docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+It's your **local** timestamp, by the way.
+
+After you provided the valid timestamp, proceed with the precompile specific configurations:
+
+```shell
+The chosen block activation time is 2023-03-31 14:00:00
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Add 'adminAddresses'?: 
+  ▸ Yes
+    No
+```
+
+The addresses added in this section allow to add other admins and/or add enabled addresses 
+for transaction issuance.
+The addresses provided in this tutorial are fake.
+
+**However, make sure you have control of the addresses respectively know who has 
+control over these addresses.
+You might prevent your Subnet to make any progress if not.**
+
+```shell
+✔ Yes
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Provide 'adminAddresses': 
+  ▸ Add
+    Delete
+    Preview
+    More Info
+↓   Done
+```
+
+The prompting runs with a pattern used throughout the tool: 
+
+* Select an operation:
+  * `Add`: adds a new address to the current list
+  * `Delete`: removes an address from the current list
+  * `Preview`: prints the current list 
+* `More info` prints additional information for better guidance, if available
+* Select `Done` when you completed the list
+
+Go ahead and add your first address:
+
+```shell
+✔ Add
+✔ Add an address: 0xaaaabbbbccccddddeeeeffff1111222233334444
+```
+
+Add another one:
+
+```shell
+✔ Add
+Add an address: 0xaaaabbbbccccddddeeeeffff1111222233334444
+✔ Add
+✔ Add an address: 0x1111222233334444aaaabbbbccccddddeeeeffff
+```
+
+Select `Preview` this time to confirm the list is correct:
+
+```shell
+✔ Preview
+0. 0xaaaAbbBBCccCDDddEeEEFFfF1111222233334444
+1. 0x1111222233334444aAaAbbBBCCCCDdDDeEeEffff
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Provide 'adminAddresses': 
+  ▸ Add
+    Delete
+    Preview
+    More Info
+↓   Done
+```
+
+If it looks good, select `Done` to continue:
+
+```shell
+✔ Done
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Add 'enabledAddresses'?: 
+  ▸ Yes
+    No
+```
+
+Add one such enabled address, these are addresses which can issue transactions:
+
+```shell
+✔ Add
+✔ Add an address: 0x55554444333322221111eeeeaaaabbbbccccdddd█
+```
+
+After you added this address, and selected `Done`, the tool asks if you want to 
+add another precompile:
+
+```shell
+✔ Done
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Should we configure another precompile?: 
+  ▸ No
+    Yes
+```
+
+If you needed to add another one, you would select `Yes` here. The wizard would guide you
+through the other available precompiles, excluding already configured ones.
+
+To not unnecessarily extend this tutorial, the assumption is you're done here.
+Select `No`, which ends the wizard.
+
+This means you have successfully terminated the generation of the upgrade file,
+often called upgrade bytes. The tool stores them internally.
+
+:::warning
+
+You shouldn't move files around manually.
+Use the `export` and `import` commands to get access to the files
+if not needed in the context of the tool
```

Comment:
```suggestion
if not needed in the context of the tool.
```
I am not fully get what this sentence means, however.

Reply:
I meant to say that the export can be used also without the tool (i.e. just manually install the file to a node).
But I think we can just safely remove that sentence altogether.